scripts/upsert_mgrb_participants.py

```diff
@@ -0,0 +1,177 @@
+import csv
+import logging
+from collections import defaultdict
+from io import StringIO
+from typing import Dict, List
+
+import click
+from google.cloud import storage
+
+from metamist.apis import ParticipantApi
+from metamist.graphql import gql, query
+from metamist.models import (
+    ParticipantUpsert,
+    SampleUpsert,
+)
+
+BUCKET_NAME = 'cpg-mgrb-test-upload'
```

Comment:
I wouldn't have hard-coded this. We're using a project name in a public repository, which is generally bad practice. We could've generalised this script to 'upsert_participants' and then passed this in at the CLI

Reply:
Also is the test-upload bucket the appropriate place at this point? Is this script for running in test or in main or both?

Reply:
We should make the bucket name as input parameter to the script.